authors/apps/authentication/serializers.py

```diff
@@ -190,6 +190,6 @@ def update(self, instance, validated_data):
         """
         Save the user profile after update
         """
-        instance.profile.save()
+        # instance.profile.save()
```

Comment:
You are supposed to delete unwanted code segments rather than commenting them out.